spec/index.html

```diff
@@ -1062,7 +1084,7 @@ <h3>The XML Schema Built-in Datatypes</h3>
       <caption>A list of the RDF-compatible XSD types, with short descriptions</caption>
       <tr><th></th><th>Datatype</th><th>Value space (informative)</th></tr>
 
-      <tr><th rowspan="4">Core types</th><td><a data-cite="xmlschema11-2#string"><code>xsd:string</code></a></td><td>Character strings (but not all Unicode character strings)</td></tr>
+      <tr><th rowspan="4">Core types</th><td><a data-cite="xmlschema11-2#string"><code>xsd:string</code></a></td><td>Character strings (but not all Unicode <a>strings</a> [[UNICODE]])</td></tr>
```

Comment:
I'm still not sure what the parenthetical is meant to do. Is it meant to call attention to the fact that `xsd:string` is restricted to `Char`?

Reply:
This is a legacy description, that I don't really understand either (maybe @afs does?) I think it could just say "Character <a>strings</a>", as I'm not clear on what strings could be represented that are not Unicode, particularly given the restriction on the XML Char production.

Reply:
I'm guessing it's the XML-Char limitation. If so, it would probably be better to just say that?

Reply:
```suggestion
      <tr><th rowspan="4">Core types</th><td><a data-cite="xmlschema11-2#string"><code>xsd:string</code></a></td><td>Character strings (see <a>string</a>)</td></tr>
```